Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,21 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+function getCardValue(checkingCard, emoji) {
+  //   const splitCard = checkingCard.split("");
+  //   return splitCard;
+  const emojiArray = ["❤️", "♠️", "♦️", "♣️"];
+  const isEmojiValid = emojiArray.includes(emoji);
+  const valueOfCard =
+    checkingCard == "A"
+      ? 11
+      : checkingCard == "K" || checkingCard == "Q" || checkingCard == "J"
+      ? 10
+      : !isNaN(checkingCard) && checkingCard >= 2 && checkingCard <= 10
+      ? Number(checkingCard)
+      : false;
+  return isEmojiValid && valueOfCard ? valueOfCard : "Invalid card rank";
```

Comment:
This part of the code does not properly handle multi-character ranks (like "10") or validate the full card format (rank + suit), and it only checks the rank and emoji separately, rather than together as a complete card.

Reply:
Sure I can do with One Parameter Pass 💯  instead of two parameter pass